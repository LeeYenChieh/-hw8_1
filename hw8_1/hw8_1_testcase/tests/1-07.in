gz1LU0oGhP5TawVeiiRbpqU cdHNvPquxDsiMPaS3txN9f4PJfLDDJANgIU
mnMTLyzG3VDRtYcrPGHgP9YWXW US5MRgsh2niVWw1NJ1t2Qam9hzL3xnstg5s39svsuYFzlXJO
pEwG8hnhRQCp8pRHrGqLlk KON
rh0WyC21gScKPRu4JLtDFUEwqbT e
mnMTLyzG3VDRtYcrPGHgP9YWXW Jfy75Y01T7Vj4rdWfsJfxZhK2dSrUKib
FFDvwHXY6f586Plx0BpwtWyJIRH luJ4XaduhkujvgeBOhqMp9TxZcxveMi6mQ8eCKEAWxb5FEMxNU031mpeu9RLts
h2rD49XdurbP qLJ8Kj5Ytyw8AjPKbbJpSuWs35KEgxWISx1BUBoRhcFaQtKfwS2RYcNdD5ruRw
gz1LU0oGhP5TawVeiiRbpqU add thickBlackBorder
Tv74qrKL8K FzdZlKPDzttYSMsKcwQo6Vn9LamUyZk9V1pipV4haDs75OcO6q
gz1LU0oGhP5TawVeiiRbpqU display
