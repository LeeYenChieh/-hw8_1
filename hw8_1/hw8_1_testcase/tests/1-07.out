cdHNvPquxDsiMPaS3txN9f4PJfLDDJANgIU thickBlackBorder 
